packages/python/plotly/plotly/tests/test_io/test_renderers.py

```diff
@@ -388,10 +388,7 @@ def test_missing_webbrowser_module(fig1):
     """
     Assert that no errors occur if the webbrowser module is absent
     """
-    try:
-        import builtins
-    except ImportError:
-        import __builtin__ as builtins
+    import builtins
```

Comment:
To make `black` happy
```suggestion
    import builtins

```